### PR TITLE
Aliases: Add aliases for arceus/others

### PR DESCRIPTION
**Pull request by TheTiksiBranch**

### CODE REVIEW

File: data/aliases.js [hunk 1]
```diff
@@ -353,6 +372,7 @@ exports.BattleAliases = {
 
 	// pokemon
 	"aboma": "Abomasnow",
+	"arc": "Arceus"
```
Comment (ascriptmaster) [inline data/aliases.js]: comma

File: data/aliases.js [hunk 2]
```diff
@@ -383,6 +403,7 @@ exports.BattleAliases = {
 	"gar": "Gengar",
 	"garde": "Gardevoir",
 	"gatr": "Feraligatr",
+	"gira": "Giratina"
```
Comment (ascriptmaster) [inline data/aliases.js]: comma here too

**Comment by TheTiksiBranch** [comment]
updated, thanks

**Comment by Zarel** [comment]
Do people use these?

**Comment by TheTiksiBranch** [comment]
From my AG experience, I know that people use mgar, arcfairy, arcghost, arcground, arcsteel, girao, phero, ygod, and dbond. I added the other arcs, arc, and gira for consistency purposes. I only listed 4 arceuses since many are near-unviable and therefore just aren't used reasonably often. I wouldn't be too opposed to removing Xurk.

**Comment by Zarel** [comment]
Fair

**Pull request merged**